src/components/HomePage/TopBanner/TopBanner.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Banner from 'components/Banner'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+
+import "./TopBanner.css"
+
+export default class TopBanner extends React.PureComponent {
+
+  render() {
+    return <Banner className="TopBanner" isClosable={true} name="dcl-top-banner">
```

Comment:
I'd change `dcl-top-banner` for something like `dcl-hackathon-banner`
Once a banner is closed the state is saved in localstorage and we won't be able to use that name again, let's keep them specific to the event just to avoid overlapping